docs/known-issues.md

```diff
@@ -1 +1,25 @@
-Placeholder - see [the WIP known issues doc](https://brevi.link/unreal-gdk-known-issues).
\ No newline at end of file
+## Unreal GDK Known Issues
+
+Known issue = any major user-facing bug or lack of user-facing feature that:
+1. diverges from vanilla Unreal design or implementation (e.g. ordering of reliable RPCs), **OR**
+1. diverges from user expectations from a SpatialOS project (e.g. interacting across worker boundaries)
+
+| Issue | Date added | Ticket | Workaround? | Done? |
+|-------|-------------------|--------|-------------|-------|
+| Order of reliable RPCs is not respected in case of unresolved UObject* parameter. | 2018-06-14 | [UNR-363](https://improbableio.atlassian.net/browse/UNR-336) | Potential user-code workaround. | No|
```

Comment:
Can we say anything more about what the user-code workaround might look like?

Reply:
@improbable-valentyn do you have any notes on what the workaround was for this issue? have the user wait for the successfull execution of the unresolved RPC before calling the second resolved one?

Reply:
I guess they would have to enforce the order explicitly in gameplay as you suggested (instead of relying on the RPCs being executed in order).

Reply:
Cool - how about "Potential user-code workaround to explicitly enforce the order of RPCs"?